backend/app/analysis/text_parser.py

```diff
@@ -0,0 +1,159 @@
+"""
+Custom command to populate text model with definition, examples, and images
+"""
+import string
+import nltk
+import Levenshtein as Lev
+
+
+# Modified Code from Bing library
+def get_sentences(text):
+    """
+    Given a Text object, stores the definitions, examples, and image urls for all the words
+    in this text.
+    :param text: the Text object that we want to get sentences from
+    """
+
+    sentences = nltk.tokenize.sent_tokenize(text.replace("\n", " "))
+
+    return sentences
```

Comment:
Looks like the docstring here might need to be updated.

Reply:
The proper docstring has been added.